source/about/cloud-dedicated.rst

```diff
@@ -0,0 +1,120 @@
+
+Encryption
+~~~~~~~~~~~
+
+Mattermost provides encryption-in-transit and encryption-at-rest capabilities. Mattermost supports :doc:`TLS encryption </install/setup-tls>`, including AES-256 with 2048-bit RSA on all data transmissions, between Mattermost client applications and the Mattermost server. You may either set up TLS on the Mattermost Server or :doc:`install a proxy such as NGINX </install/proxy-to-mattermost-transport-encryption>`, and set up TLS on the proxy.
+
+Connections to :doc:`Active Directory/LDAP </onboard/ad-ldap>` can :ref:`optionally be secured with TLS or stunnel <configure/authentication-configuration-settings:ad/ldap port>`.
+
+Connections to calls are secured with a combination of:
+
+- TLS: The existing WebSocket channel is used to secure the signaling path.
+- DTLS v1.2 (mandatory): Used for initial key exchange. Supports ``TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256`` and ``TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA`` algorithms.
+- SRTP (mandatory): Used to encrypt all media packets (i.e. those containing voice or screen share). Supports ``AEAD_AES_128_GCM`` and ``AES128_CM_HMAC_SHA1_80`` algorithms.
+
+Gossip encryption (experimental)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost supports encryption of cluster data in-transit when using the gossip protocol. See our :ref:`gossip encryption <deploy/encryption-options:gossip encryption (experimental)>` documentation for details.
+
+The encryption uses AES-256 by default, and it is not configurable. However, it is possible to manually set the value in the ``Systems`` table for the ``ClusterEncryptionKey`` row. A key is a byte array converted to ``base64``. It can be set to a length of 16, 24, or 32 bytes to select ``AES-128``, ``AES-192``, or ``AES-256`` respectively.
+
+Bring your own key encryption
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost Cloud Dedicated customers can ensure autonomy over their encryption key life cycle with :doc:`Bring Your Own Key (BYOK) encryption </manage/cloud-byok>`. BYOK supports encryption at rest with custom KMS keys that the enterprise provides and maintains. 
+
+You can use KMS keys in 2 ways:
+
+- One KMS key for all services; or,
+- Per-service KMS keys (EBS, RDS, S3)
+
+  - Keys do not need to be unique to each service.
+  - All services must be encrypted at rest.
+  - Selective enablement of this feature can be supported.
+  - In cases where a global database is needed, we recommend providing 2 KMS keys (1 per region).
+
+SMTP
+~~~~
+
+Email sent from Mattermost Cloud Dedicated uses XXX, and the connection to XXX is encrypted. See our :doc:`SMTP </configure/smtp-email>` documentation for setup details.
+
+Audit and observability
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost Cloud Dedicated provides access to :doc:`audit and system logs </manage/logging>` generated by the application. 
+
+Bring your own cloud
```

Comment:
I am not sure if we should include this at this moment. @nab-77 what do you think?

Reply:
Removed for now; can re-add in a later iteration.